src/Formats/KafkaSchemaRegistry.h

```diff
@@ -14,14 +16,57 @@ class KafkaSchemaRegistry final
 public:
     static UInt32 readSchemaId(ReadBuffer & in);
 
+    /// The key type for caching KafkaSchemaRegistry
+    struct CacheKey
+    {
+        String base_url;
+        String credentials;
+        String private_key_file;
+        String certificate_file;
+        String ca_location;
+        bool skip_cert_check;
+
+        bool operator ==(const CacheKey & rhs) const
```

Comment:
clang-format

Reply:
Never mind, I need to rebase the branch anyways, will fix it.